autoload/OmniSharp/actions/members.vim

```diff
@@ -74,20 +74,20 @@ function! s:ComputeItemSignature(item) abort
   if textBeforeDisplayName !~# '^\(private\|internal\|protected\|public\)'
     let textBeforeDisplayName = a:item.Properties.accessibility . ' ' . textBeforeDisplayName
   endif
-  return ReduceToOneCharacter(textBeforeDisplayName) . a:item.DisplayName
+  return s:ReduceToOneCharacter(textBeforeDisplayName) . a:item.DisplayName
 endfunction
 
-let s:SingleCharacterSymbolByAccessModifier = {
- \ 'public': '+',
- \ 'private': '-',
- \ 'internal': '&',
- \ 'protected': '|'
-\}
```

Comment:
When this dictionary is defined as a script-level variable, it only needs to be defined once. When it is defined as a local variable inside a function, it is defined every time the function is called which creates unnecessary overhead.

It is very little overhead, but I don't think it makes sense to move it this way.